### PR TITLE
Mpu6050 driver fixed

### PR DESCRIPTION
**Pull request by vizhyk**
@yekovalyov Please, review my task. Don't pay attention to first two commits. I should to clean master, but will do later to avoid troubles with hometask's commits .

### CODE REVIEW

File: mpu6050.c [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+		TEMP_OUT_L = i2c_smbus_read_byte_data(drv_client, REG_TEMP_OUT_L);
+		TEMP_OUT_H = i2c_smbus_read_byte_data(drv_client, REG_TEMP_OUT_H);
+
+		temp = (TEMP_OUT_H << 8) | TEMP_OUT_L;
+
+		integer += ((temp + 12420) * 9 / 1700 + 32);
+		rest += ((temp + 12420) * 90 / 17) % 1000;
+
+
+		while (rest >= 1000) {
```
Comment (yekovalyov) [inline mpu6050.c]: Please implement conversation to integer and fractional part without additional loops.

File: mpu6050.c [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+	dev_info(&drv_client->dev, "sensor data read:\n");
+	dev_info(&drv_client->dev, "ACCEL[X,Y,Z] = [%d, %d, %d]\n",   //changed sudev into dev
+		g_mpu6050_data.accel_values[0],
+		g_mpu6050_data.accel_values[1],
+		g_mpu6050_data.accel_values[2]);
+	dev_info(&drv_client->dev, "GYRO[X,Y,Z] = [%d, %d, %d]\n",
+		g_mpu6050_data.gyro_values[0],
+		g_mpu6050_data.gyro_values[1],
+		g_mpu6050_data.gyro_values[2]);
+	dev_info(&drv_client->dev, "TEMP = [%d.%d F]\n",
```
Comment (yekovalyov) [inline mpu6050.c]: This format %d.%d will give wrong results. Please explain me why.

**Review by yekovalyov — CHANGES_REQUESTED** [review]

File: mpu6050.c (hunk 2 above)
Comment (vizhyk) [reply mpu6050.c]: My temperature output consist of two integer values, that's why was used %d%d.
Temperature output looks correct or am I wrong?

Comment (vizhyk) [reply mpu6050.c]: Screenshot from 2019-05-10 17-55-05

Comment (yekovalyov) [reply mpu6050.c]: You hadn't tested enough your solution. Please repeat the same test at least ones again.

Comment (yekovalyov) [reply mpu6050.c]: What will happen if fractional part will be less than 100?

Comment (yekovalyov) [reply mpu6050.c]: Also please fix errors and warnings in your code produced by linux/scripts/checkpatch.pl --no-tree -f *.c

Comment (vizhyk) [reply mpu6050.c]: @yekovalyov I have fixed temperature output and code style seems good in checkpatch utility.

**Review by yekovalyov — APPROVED** [review]

**Pull request merged**